packages/shell-chrome/views/_partials/footer.edge

```diff
@@ -0,0 +1,109 @@
+<div
+    :class="{
+    'hidden': !showTools,
+}"
+    class="flex font-bold text-gray-400 border-t border-gray-300 bg-white"
+>
+    <div class="flex-1">
+        <div class="flex items-center text-xs leading-9 font-medium font-mono">
+            <div class="flex-1 pl-3" x-text="footerText"></div>
```

Comment:
should we stick a `data-testid` on this and add an assertion in the `component.js` test suite that it displays the right text?

Reply:
For sure. Done.